pkgs/applications/networking/cluster/waagent/default.nix

```diff
@@ -1,25 +1,18 @@
-{ fetchFromGitHub,
-  findutils,
-  gnugrep,
-  gnused,
-  iproute2,
-  iptables,
-  lib,
-  nettools, # for hostname
-  openssh,
-  openssl,
-  parted,
-  procps, # for pidof,
-  python39, # the latest python version that waagent test against according to https://github.com/Azure/WALinuxAgent/blob/28345a55f9b21dae89472111635fd6e41809d958/.github/workflows/ci_pr.yml#L75
-  shadow, # for useradd, usermod
-  util-linux, # for (u)mount, fdisk, sfdisk, mkswap
+{ bash
+, coreutils
+, fetchFromGitHub
+, lib
+  # the latest python version that waagent test against according to https://github.com/Azure/WALinuxAgent/blob/28345a55f9b21dae89472111635fd6e41809d958/.github/workflows/ci_pr.yml#L75
+, python39
+, substituteAll
 }:
 
 let
   inherit (lib) makeBinPath;
+  python = python39;
```

Comment:
Any reason why `python39` is sometimes referred to directly instead of as `python` within this file?

Reply:
Ugh, I wanted to replace all occurences with `python`, and leave the detail it's 3.9 to that let binding - and forgot to update the references. PTAL.